cmake/defaults.cmake

```diff
@@ -0,0 +1,16 @@
+# Default values for all projects that can be set before the project() command
+# No `if()` statements allowed in here, particularly anything that's predecated
+# on platform or compiler detection
+
+set_property(GLOBAL PROPERTY USE_FOLDERS ON)
+set_property(GLOBAL PROPERTY PREDEFINED_TARGETS_FOLDER "CMakeTargets")
+
+set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
Just to confirm, cesium-native intends to stay on C++ 17 for now? (as opposed to C++ 20)

Reply:
Yes, for now that's intentional.